modules/util/portal-tools-service-builder-test-test/src/testIntegration/java/com/liferay/portal/tools/service/builder/test/service/persistence/impl/test/PersistenceNestedSetsTreeManagerTest.java

```diff
@@ -61,10 +62,13 @@
 public class PersistenceNestedSetsTreeManagerTest {
 
 	@ClassRule
+	@Rule
 	public static final AggregateTestRule aggregateTestRule =
 		new AggregateTestRule(
-			CodeCoverageAssertor.INSTANCE, new LiferayIntegrationTestRule(),
-			TransactionalTestRule.INSTANCE);
+			new LiferayIntegrationTestRule(),
```

Comment:
@dantewang You are removing the CodeCoverageAssertor one, this is changing the logic. Is it possible that we add CodeCoverageAssertor support for Arquillian test? There must be other integration tests using CodeCoverageAssertor